flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManagerImplTest.java

```diff
@@ -753,33 +742,18 @@ public void testTaskManagerSlotRequestTimeoutHandling() throws Exception {
 
 			slotManager.registerSlotRequest(slotRequest);
 
-			ArgumentCaptor<SlotID> slotIdCaptor = ArgumentCaptor.forClass(SlotID.class);
-
-			verify(taskExecutorGateway, times(1)).requestSlot(
-				slotIdCaptor.capture(),
-				eq(jobId),
-				eq(allocationId),
-				anyString(),
-				eq(resourceManagerId),
-				any(Time.class));
-
-			TaskManagerSlot failedSlot = slotManager.getSlot(slotIdCaptor.getValue());
+			final SlotID firstSlotId = slotIds.take();
```

Comment:
this isn't a 1:1 port since the size check is missing

Reply:
will add the empty check